Rakefile

```diff
@@ -602,14 +602,51 @@ namespace :changelog do
   end
 end
 
-Rake::ExtensionTask.new("ddtrace_profiling_native_extension.#{RUBY_VERSION}_#{RUBY_PLATFORM}") do |ext|
+native_exts = []
```

Comment:
Minor: I would suggest making this into a constant, to make it clear that it's in the global scope.

(Note that the array itself pointed to by the constant is still mutable, so everything else would work fine as-is)

Reply:
Needed to change things a little bit because rubocop doesn't really like unfrozen constants but the end result is what you suggested 👍